docs/contribute.rst

```diff
@@ -84,6 +84,11 @@ everything needed for the tmt development ready on your system::
 
     pip install '.[all]'
 
+Install ``pre-commit`` to run all available checks before commit::
+
+    sudo dnf -y install pre-commit
```

Comment:
maybe we should use `pip` installation instead?

Reply:
I think you can mention both and people can decide what they prefer

Reply:
added

Reply:
Everything else in this section assumes using a virtual environment. I'd say we don't need to mention other alternatives,  `pip` is enough. Perhaps we should add `pre-commit` to `setup.py` extra requires so that `pip install '.[tests]'` brings everything needed for testing new code?

Reply:
not a bad idea, changing